c-realV2.py

```diff
@@ -2328,7 +2328,8 @@ def wrapper(self, *args, **kwargs):
         exit()
     client.run(settings['token'])
 except discord.PrivilegedIntentsRequired:
-    print('PrivilegedIntentsRequired: This field is required to request for a list of members in the discord server that the bot is connected to. Watch https://youtu.be/DXnEFoHwL1A?t=44 to see how to turn on the required field.')
+    print(chr(27) + "[2J")
+    print('PrivilegedIntentsRequired: Watch https://youtu.be/DXnEFoHwL1A?t=44 to see how to turn on the required field.')
```

Comment:
This tutorial is a little outdated, maybe use a newer YouTube video? or maybe link it to step 8 of the [How to get the bot token?](https://github.com/TKperson/Nuking-Discord-Server-Bot-Nuke-Bot/wiki/Basic-setup-and-knowledge-for-using-the-bot#how-to-get-the-bot-token)

Reply:
alr, i can make one by tomr or something ill hop on a vm